tools/capable.py

```diff
@@ -182,7 +182,15 @@ def __getattr__(self, name):
     }
 
     u32 uid = bpf_get_current_uid_gid();
-    struct data_t data = {.tgid = tgid, .pid = pid, .uid = uid, .cap = cap, .audit = audit, .insetid = insetid};
+
+    struct data_t data = {};
```

Comment:
The C standard doesn't enforce compiler to initialize all memory occupied with the structure with 0. It only dictates all named fields to be initialized with 0 and all padding value are undefined and the compiler has the right leaving them with any value. But typically clang compiler will try to initialize the whole thing. So let us keep this change until/if it breaks.